app/scenes/Document/components/Header.js

```diff
@@ -274,4 +282,19 @@ const Status = styled(Action)`
   color: ${(props) => props.theme.slate};
 `;
 
+const TocWrapper = styled(Action)`
+  padding-left: 0;
+  padding-right: 4px;
+  color: ${(props) => props.theme.slate};
+  position: fixed;
+  margin-left: 8px;
+  top: 12px;
+  left: 46px;
+  z-index: ${(props) => props.theme.depths.sidebar - 1};
```

Comment:
On mobile rn so I can't check - (screenshot?) but all of these positioning styles should be covered by using the context menu

Reply:
Here is the screenshot. Please take a look.

![Screen Shot 2021-07-08 at 09 09 06](https://user-images.githubusercontent.com/40148448/124859158-fd3eb600-dfcc-11eb-8fbb-7f3a192e9f18.png)

> but all of these positioning styles should be covered by using the context menu

I couldn't do it. Let me know how I should go about it.

Reply:
Looks like if you move it inside `Header.actions` it becomes easier to position – can absolute position then:

```
position: absolute;
left: 52px;
```

Reply:
99% sure it's unrelated, but do you see horizontal scrolling on docs on mobile when testing this?

Reply:
> Looks like if you move it inside `Header.actions` it becomes easier to position – can absolute position then:
> 
> ```
> position: absolute;
> left: 52px;
> ```

Worked like a charm. 

> you see horizontal scrolling on docs on mobile when testing this?

No, I didn't. Sometimes it was there when I was manually resizing the window, but a hard refresh fixes everything, and I couldn't find any overflowing on the predefined mobile resolutions of browsers.